source/fem-dem/vans_assemblers.cc

```diff
@@ -1289,34 +1282,32 @@ void
 VANSAssemblerGidaspow<dim>::calculate_particle_fluid_interactions(
   NavierStokesScratchData<dim> &scratch_data)
 {
-  // particle_number is an increment that goes from 0 to n_particles_in_cell.
-  // It is incremented at the end of the loop over particles and is used to
-  // point to the element of the vectors relative_velocity and
-  // fluid_velocity_at_particle_location corresponding to the particle being
-  // looped over.
-  double      cell_void_fraction = 0;
-  const auto &relative_velocity =
-    scratch_data.fluid_particle_relative_velocity_at_particle_location;
-  const auto &Re_p      = scratch_data.Re_particle;
-  auto       &beta_drag = scratch_data.beta_drag;
-
-  Tensor<1, dim> drag_force;
-
-
   // Physical Properties
   Assert(
     !scratch_data.properties_manager.is_non_newtonian(),
     RequiresConstantViscosity(
       "VANSAssemblerGidaspow<dim>::calculate_particle_fluid_interactions"));
-  const double kinematic_viscosity =
-    scratch_data.properties_manager.get_kinematic_viscosity_scale();
 
   Assert(
     scratch_data.properties_manager.density_is_constant(),
     RequiresConstantDensity(
       "VANSAssemblerGidaspow<dim>::calculate_particle_fluid_interactions"));
-  const double density = scratch_data.properties_manager.get_density_scale();
 
+  // particle_number is an increment that goes from 0 to n_particles_in_cell.
+  // It is incremented at the end of the loop over particles and is used to
+  // point to the element of the vectors relative_velocity and
+  // fluid_velocity_at_particle_location corresponding to the particle being
+  // looped over.
```

Comment:
there is no variable named particle number near here.... I would consider moving this comment to a more appropriate location.